src/Whoops/Handler/JsonResponseHandler.php

```diff
@@ -21,6 +21,26 @@ class JsonResponseHandler extends Handler
     private $returnFrames = false;
 
     /**
+     * @var bool
+     */
+    private $jsonApi = false;
+
+    /**
+     * @param bool|false 
+     * @return bool|status
+     */
+    public function setJsonApi($jsonApi=false)
+    {
+
+      if (is_bool($jsonApi)) {
+        $this->jsonApi = $jsonApi;
+        return true;
+      }
+
+      return false;
+    }
```

Comment:
/**
     * @param bool Default is false
     * @return $this
     */
    public function setJsonApi($jsonApi = false)
    {

        $this->jsonApi = (bool) $jsonApi;

        return $this;
    }
is less code and do the same thing